src/pages/Lending/Pool/Pool.tsx

```diff
@@ -48,11 +47,14 @@ import { Faq } from './components/Faq'
 import { PoolInfo } from './components/PoolInfo'
 import { Repay } from './components/Repay/Repay'
 
-const containerPadding = { base: 6, '2xl': 8 }
 const tabSelected = { color: 'text.base' }
 const maxWidth = { base: '100%', md: '450px' }
 const responsiveFlex = { base: 'auto', lg: 1 }
-const PoolHeader = () => {
+
+type PoolHeaderProps = {
+  assetName?: string
```

Comment:
Non-blocking, would just pass `assetId`, select the asset and access asset.name here